python/setup.cfg

```diff
@@ -1,2 +1,42 @@
-# This file contains the default option values to be used during setup. An
-# example can be found at https://github.com/pypa/sampleproject/blob/master/setup.cfg
+[metadata]
+name = graphframes
+version = 0.8.5
+description = GraphFrames: Graph Processing Framework for Apache Spark
+long_description = file: ../README.md
+long_description_content_type = text/markdown
+author = GraphFrames Contributors
+author_email = graphframes@googlegroups.com
+url = https://pypi.org/project/graphframes-py/
+license = Apache License 2.0
+classifiers =
+    Development Status :: 4 - Beta
+    Programming Language :: Python :: 3
+    Operating System :: OS Independent
+
+[options]
+packages = find:
+package_dir =
+    = python
+include_package_data = True
+install_requires =
+    pyspark>=2.0.0
+    click==8.1.8
```

Comment:
Why do we need it in main requirements?

Reply:
Because `graphframes.tutorials.download` uses it. `Click` has no dependencies except for `colorama` on Windows, so it seems a safe inclusion. If you want me to pull it into an extras, I think that could confuse new users who try to run the tutorials...